### ISSUE TITLE
Text selection: Mouse double click should select a word, and allow highlighting by words

### ISSUE THREAD

**Issue by Ciantic** [linebender/druid#1570]
I tested on TextBox with Windows 10, following didn't work:

- [ ] Mouse double click on word should select a word
- [ ] Mouse double click on a word, *and* leave mouse button pressed, and then moving mouse selects text by words

Not sure how these are implemented in other OS's but that is a common in Windows 10.

Additionally there is sometimes:

- [ ] Mouse triple click selects whole text, e.g. Google Chrome address bar

**Comment by cmyr** [comment]
Agree, this is an oversight. I'm not sure how it will interact with IME, but that's something to keep in mind. The behaviours you describe are also standard on macOS.

**Comment by ngugcx** [comment]
In the flex example of the master branch, a word is still not selected in the text box after double-clicking the mouse.

**Comment by cmyr** [comment]
@ngugcx what platform are you on?

**Comment by ngugcx** [comment]
Windows 10. @cmyr
